lib/bloc/currency/currency_bloc.dart

```diff
@@ -28,7 +25,7 @@ class CurrencyBloc extends Cubit<CurrencyState> with HydratedMixin {
   }
 
   void listFiatCurrencies() {
-    _breezSDK.listFiatCurrencies().then((fiatCurrencies) {
+    liquidSdk.wallet!.listFiatCurrencies().then((fiatCurrencies) {
```

Comment:
I know this isn't under review (sorry for throwing the 🔧). I think name `wallet` might become confusing for devs using the SDK, especially when listing fiat rates etc. They might assume its access to the underlying liquid wallet. Something like `instance` might be better (if it's not a reserved word) `liquidSdk.instance!.listFiatCurrencies()`

Reply:
Sounds good. Let's open an issue for it.

Reply:
Done